qt/python/mantidqt/widgets/plotconfigdialog/legendtabwidget/presenter.py

```diff
@@ -39,7 +39,7 @@ def __init__(self, fig, view=None, parent=None):
         self.view.advanced_options_push_button.clicked.connect(self.show_advanced_options)
         self.view.advanced_options.rejected.connect(self.advanced_options_cancelled)
 
-    def init_view(self):
+    def update_view(self):
```

Comment:
This should still be called `init_view` looking at the implementation, as lines like
```
        if int(matplotlib.__version__[0]) < 2:
            self.view.hide_box_properties()
```
Should never need to be re-run for example

Any parts which update the GUI should be moved into a new method called `update_view` then called at the end of init instead